lib/import_expense_screen.dart

```diff
@@ -170,7 +179,10 @@ class _ImportExpensePageState extends State<ImportExpensePage> {
                       decoration: customUnderlineInputdecoration(
                         hintText: 'Enter Name or select from contact',
                         bordersideColor: primaryColor,
-                        suffixicon: customContactUi(onTap: _contactFetchFn),
+                        suffixicon: customContactUi(onTap:(){
```

Comment:
I dont think this code is needed as you are repeating this in `customContactUi(..)`